apps/portals/elportal/src/themes/HomePageThemeProvider.tsx

```diff
@@ -14,6 +14,20 @@ export function HomePageThemeProvider({ children }: React.PropsWithChildren) {
         fontFamily: "'Merriweather', serif",
       },
     },
+    components: {
+      MuiButton: {
+        styleOverrides: {
+          contained: {
+            animationTimingFunction: 'ease-out',
+            animationDuration: '64ms',
+            '&:hover': {
+              color: '#FFF !important',
```

Comment:
I think this color is only overridden in the individual component instance because you're using a react-router link, which makes this an `a` tag, and we have Bootstrap 3 in the project which gives all a:hover the wrong color.

Could you move this specific style to the button component instance, and add a comment like
```ts
// TODO: Bootstrap 3 link color applies to all a:hover elements. Remove this override when Bootstrap 3 is removed
```